plugins/by-name/nvim-ufo/default.nix

```diff
@@ -130,4 +132,16 @@ lib.nixvim.plugins.mkNeovimPlugin {
         end
       '';
   };
+
+  extraConfig = cfg: {
+    plugins.lsp.capabilities =
+      lib.mkIf cfg.settings.setupLspCapabilities # lua
```

Comment:
```suggestion
      lib.mkIf cfg.setupLspCapabilities # lua
```

Reply:
Done by putting the option in `extraOptions` :+1: